quipucords/fingerprinter/jboss_eap.py

```diff
@@ -3,7 +3,7 @@
 import bisect
 import logging
 
-from api.models import Product
+from api.deployments_report.model import Product
```

Comment:
outside of `api` package we should be importing all models from `api.models`. Please change it back.